src/FsUnit.Xunit/FsUnitTyped.fs

```diff
@@ -0,0 +1,61 @@
+namespace FsUnitTyped
+
+open System.Diagnostics
+open FsUnit.Xunit
+
+[<AutoOpen>]
+module TopLevelOperators =
+
+    /// Asserts that `expected` is equal to `actual`.
+    /// The equality instance on `expected` is used, if available.
+    [<DebuggerStepThrough>]
+    let shouldEqual<'a> (expected: 'a) (actual: 'a) =
+        should equal expected actual
+
+    /// Asserts that `expected` is not equal to `actual`.
+    /// The equality instance on `expected` is used, if available.
+    [<DebuggerStepThrough>]
+    let shouldNotEqual<'a> (expected: 'a) (actual: 'a) =
+        should not' (equal actual) expected
```

Comment:
`actual` and `exoected` swapped here as well

Reply:
Yes, this was what happened when I was trying to fix a failing test - but it turned out not to be because I'd got the arguments in the wrong order, but instead because of [the inconsistent behaviour of `should equal` between NUnit and Xunit](https://github.com/fsprojects/FsUnit/pull/203#issuecomment-1087226133).